packages/libs/wdk-client/src/Views/Question/Params/StringParam.tsx

```diff
@@ -25,22 +25,23 @@ function Component(props: Props<StringParam, undefined>) {
   const { parameter, value, onParamValueChange } = props;
   const cols = DEFAULT_COLS;
   const rows = calculateRows(parameter, cols);
-  return parameter.length <= 50 ? (
-    <TextBox
-      type="text"
-      value={value}
+  return parameter.isMultiLine ? (
+    <TextArea
+      cols={cols}
+      rows={rows}
       readOnly={parameter.isReadOnly}
+      value={value}
       onChange={onParamValueChange}
       required={!parameter.allowEmptyValue}
     />
   ) : (
-    <TextArea
-      cols={cols}
-      rows={rows}
-      readOnly={parameter.isReadOnly}
+    <TextBox
+      type="text"
       value={value}
+      readOnly={parameter.isReadOnly}
       onChange={onParamValueChange}
       required={!parameter.allowEmptyValue}
+      size={40}
```

Comment:
If `parameter.isNumber` is `true`, set this to `20`.

Reply:
I realized that I never made it clear that `StringParam` can also be used for numbers, even though we do have a `NumberParam`. It's possible `isNumber` is not used anymore.